contracts/utils/convex/ConvexStakingWrapper.sol

```diff
@@ -308,15 +271,23 @@ contract ConvexStakingWrapper is ERC20 {
     /// @return claimable Array of earned tokens and their amount
     function earned(address _account) external view returns (EarnedData[] memory claimable) {
         uint256 supply = _getTotalSupply();
+        // uint256 depositedBalance = _getDepositedBalance(_account);
         uint256 rewardCount = rewards.length;
-        claimable = new EarnedData[](rewardCount + 1);
+        claimable = new EarnedData[](rewardCount);
 
-        for (uint256 i; i < rewardCount; ++i) {
+        for (uint256 i = 0; i < rewardCount; i++) {
```

Comment:
No need to initialize `i` if it is zero.

In my implementation I initialized it to `1`, and did the `cvx` block before the loop. Now I see that I might be missing something.

Reply:
If we do this then if there is any new claimable reward that is there for `cvx` would be missed. The one calculated [here](https://github.com/yieldprotocol/vault-v2/blob/45256b8aca37b23b306b2a09666a5e324e14e0a9/contracts/utils/convex/ConvexStakingWrapper.sol#L299)